pkg/models/registries/image_test.go

```diff
@@ -0,0 +1,83 @@
+package registries
+
+import (
+	"testing"
+)
+
+func TestParseImage(t *testing.T) {
+	type image struct {
+		ImageName string
+		ExDomain  string
+		ExTag     string
+		ExPath    string
```

Comment:
this is not a good unit test example, you should define what expected result would like, and compare expected result with what you got using reflect